docs/products/mysql/howto/migrate-database-mysqldump.rst

```diff
@@ -0,0 +1,122 @@
+
+The below example uses Aiven for MySQL databases for both, ``source-db`` and ``target-db``. You can create the databases by going to your running service for **Aiven for MySQL** > **Databases** > **Create a new database**.
+
+
+Backup the data
+---------------
+
+Variables
+'''''''''
+
+To backup the ``source-db`` data to a file called ``mydb_backup.sql`` you need to collect some information about your Aiven for MySQL ``source-db`` database. Go to your Aiven ``source-db`` service, from **Overview** page, find the following information:
+
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Variable
+     - Description
+   * - ``SOURCE_DB_HOST``
+     - **Host** name for the connection
+   * - ``SOURCE_DB_USER``
+     - **User** name for the connection
+   * - ``SOURCE_DB_PORT``
+     - Connection **Port** number
+   * - ``SOURCE_DB_PASSWORD``
+     - Connection **Password**
+   * - ``DEFAULTDB``
+     - Database that contains the ``source-db`` data
+
+Commands
+'''''''''
+
+Use the following command to backup your Aiven for MySQL data to the ``mydb_backup.sql`` file:
+
+.. code-block:: shell
+
+    mysqldump \
+    -p DEFAULTDB -P SOURCE_DB_PORT \
+    -h SOURCE_DB_HOST --single-transaction \
+    -u SOURCE_DB_USER --set-gtid-purged=OFF \
+    --password > mydb_backup.sql
+
+
+With this command, the password will be requested at the prompt; paste ``SOURCE_DB_PASSWORD`` to the terminal, then a file named ``mydb_backup.sql`` will be created with your backup data. Note that having the prompt request for the password is more secure than including the password straight away in the command. 
+
+The ``--single-transaction`` `flag <https://dev.mysql.com/doc/refman/8.0/en/mysqldump.html#option_mysqldump_single-transaction>`_ starts a transaction in isolation mode ``REPEATABLE READ`` before running, allowing ``mysqldump`` to read the database in the current state at the time of the transaction and therefore ensuring consistency of the data.
+
+.. warning::
+
+  If you are using `Global Transaction Identifiers <https://dev.mysql.com/doc/refman/5.7/en/replication-gtids-concepts.html>`_ (GTIDs) with InnoDB use the ``--set-gtid-purged=OFF`` `option <https://dev.mysql.com/doc/refman/8.0/en/mysqldump.html#option_mysqldump_set-gtid-purged>`_. The reason is that GTID's are not available with MyISAM.
+
+
+
+  
+
+
+
+
+
+Restore the data
+----------------
+
+Variables
+'''''''''
+
+You can back up the data previously saved in a file to an Aiven for MySQL database. You need to collect some information about your Aiven for MySQL ``target-db`` to restore the data. Go to your Aiven ``target-db`` service, from **Overview** page, find the following information:
+
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Variable
+     - Description
+   * - ``TARGET_DB_HOST``
+     - Service host name
```

Comment:
Same comment as above. Either way, since it is the same information we are getting from the Console it would be good to have the naming/formatting the same in both tables.

Reply:
I think this is a great idea, I changed here: https://github.com/aiven/devportal/pull/1345/commits/d70215aec71ee82736fb18c6f40d5430742ee6b6